documentation/identifierReport.md

```diff
@@ -0,0 +1,29 @@
+# Report: identifier
+
+## Description
+
+- `publicationId`
+  - type: URI
+  - description: The resource uri of the publication
+  - example: <https://api.nva.unit.no/publication/{sortableIdentifier}>
+- `status`
+  - type: string
+  - enum: [PUBLISHED, UNPUBLISHED, DELETED]
+  - description: The status of the publication
+  - example: PUBLISHED
+- `publicationIdentifier`
+  - type: string
+  - description: The sortable identifier of the publication
+  - example: 018a8931a9d8-910a8ae0-77fd-4dba-ab12-efe82279b44e
+- `fundingSource`
```

Comment:
This seems to be a copy of the fundingReport. You probably need to adjust it to contain identifiers.

Reply:
This seems weird, but it is actually how the identifier-report is set up today. Analyseplattformen asked the same question. I will report a bug and update the identifier report (with this documentation)